diff -Nurp aalib-1.2/aalib.h aalib-1.2-typo_fix/aalib.h
--- aalib-1.2/aalib.h	Tue Nov 11 20:06:44 1997
+++ aalib-1.2-add_/aalib.h	Mon Sep 14 20:53:53 1998
@@ -183,7 +183,7 @@ typedef struct aa_context aa_context;
 typedef struct aa_linkedlist aa_linkedlist;
 typedef struct aa_linkedlist aa_reclist;
 typedef struct aa_renderparams aa_renderparams;
-typedef struct aa_hardwareparams aa_hardware_params;
+typedef struct aa_hardware_params aa_hardware_params;
 typedef struct aa_driver aa_driver;
 typedef struct aa_font aa_font;
 typedef struct aa_format aa_format;
